assets/js/blocks/classic-template/utils.ts

```diff
@@ -31,6 +31,10 @@ export function getTemplateDetailsBySlug(
 		}
 	}
 
+	if ( parsedTemplate.includes( 'taxonomy-product_' ) ) {
```

Comment:
Would it be more appropriate to use `startsWith` instead of `includes`?

```suggestion
	if ( parsedTemplate.startsWith( 'taxonomy-product_' ) ) {
```